main.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const fastify = require('fastify');
+const pinoLogger = require('pino');
```

Comment:
There is no need to require pino here.
* Either pass custom stream to fastify
* Or create custom pino logger wrapper in /src/logger and pass it to both fastify and sandbox

Reply:
Done